x-pack/solutions/security/plugins/security_solution/public/data_view_picker/components/data_view_picker/index.tsx

```diff
@@ -0,0 +1,131 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { DataViewPicker as UnifiedDataViewPicker } from '@kbn/unified-search-plugin/public';
+import React, { useCallback, useRef, useMemo, memo } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+
+import { DataView, type DataViewListItem } from '@kbn/data-views-plugin/public';
+import type { DataViewPickerScopeName } from '../../constants';
+import { useKibana } from '../../../common/lib/kibana';
+import { DEFAULT_SECURITY_SOLUTION_DATA_VIEW_ID } from '../../constants';
+import { useDataView } from '../../hooks/use_data_view';
+import { sharedStateSelector } from '../../redux/selectors';
+import { shared } from '../../redux/slices';
+import { useSelectDataView } from '../../hooks/use_select_data_view';
+
+export const DataViewPicker = memo((props: { scope: DataViewPickerScopeName }) => {
+  const dispatch = useDispatch();
+  const selectDataView = useSelectDataView();
+
+  const {
+    services: { dataViewEditor, data, dataViewFieldEditor, fieldFormats },
+  } = useKibana();
+
+  const closeDataViewEditor = useRef<() => void | undefined>();
+  const closeFieldEditor = useRef<() => void | undefined>();
+
+  // TODO: this should be disabled for the default data views probably, eg. `security-solution-default`
```

Comment:
It _should_ be, but it can be done from the Discover UI...though it doesn't actually seem to affect the advanced setting which we read from (so I think a new DV is created there..)